lms/djangoapps/bookmarks/serializers.py

```diff
@@ -0,0 +1,44 @@
+"""
+Serializers for Bookmarks.
+"""
+from rest_framework import serializers
+
+from .models import Bookmark
+
+
+class BookmarkSerializer(serializers.ModelSerializer):
+    """
+    Class that serializes the Bookmark model.
+    """
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'fields' arg up to the superclass
+        try:
+            fields = kwargs['context'].pop('fields', [])
+        except KeyError:
+            from .views import DEFAULT_FIELDS
```

Comment:
I assume this is done this way to avoid circular imports. Can you put DEFAULT_FIELDS in the init file so this is not an issue?